cost/azure/rightsize_compute_instances/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v3.0
```

Comment:
V1.0 - explained better on another comment but happy to discuss further with the team (Michael, Lawrence, Shawn, etc..)

Reply:
Updated to v1.0